### PR TITLE
Change weblink working copy methods

### PR DESCRIPTION
**Pull request by brissons**
## Description

Update naming for working copy stuff for weblinks, since they were updated [in this PR](https://github.com/BrightspaceHypermediaComponents/siren-sdk/pull/360).

## Goal/Solution

Some code cleanup.

## Related PRs

https://github.com/BrightspaceHypermediaComponents/siren-sdk/pull/360

## Note

We will need to merge the related PR first for the tests to pass.

### CODE REVIEW

**Review by ianlaidlaw — APPROVED** [review]

**Review by coffeexcode — DISMISSED** [review]

**Review by CodeBaboon — DISMISSED** [review]
OYBT

**Review by ChrisKraljevicD2L — APPROVED** [review]

**Comment by ChrisKraljevicD2L** [comment]
[hmmmm](https://github.com/BrightspaceHypermediaComponents/activities/pull/1879/checks?check_run_id=3062903868#step:6:233) does this PR need to pull in the new release of siren-sdk that you cut Scott?

**Review by ianlaidlaw — APPROVED** [review]

File: test/d2l-activity-editor/d2l-activity-content-editor/state/content-web-link.spec.js [hunk 1]
```diff
@@ -31,7 +31,7 @@ describe('Content Web Link', function() {
 				isExternalResource: () => false,
 				self: () => 'http://test-web-link-href.com',
 				getActivityUsageHref: () => 'http://test-activity-usage-link-href.com',
-				checkoutWebLink: checkoutWebLinkMock
```
Comment (ianlaidlaw) [inline test/d2l-activity-editor/d2l-activity-content-editor/state/content-web-link.spec.js:34]: 🤪

Comment (brissons) [reply test/d2l-activity-editor/d2l-activity-content-editor/state/content-web-link.spec.js:34]: @ChrisKraljevicD2L 🤪

**Pull request merged**